elyra/contents/handlers.py

```diff
@@ -38,28 +38,22 @@ async def get(self, path=''):
         path = path or ''
         path = url_unescape(path)
 
-        type = self.get_query_argument('type', default=None)
-        if type not in {None, 'file', 'notebook'}:
-            raise web.HTTPError(400, u'Type %r is invalid' % type)
-
         self.log.debug("Parsing file: %s", path)
 
         try:
             model = await self.operation_parser(path)
         except FileNotFoundError as fnfe:
             model = dict(title=str(fnfe))
             self.set_status(404)
+            raise web.HTTPError(404, str(fnfe)) from fnfe
         except IsADirectoryError as iade:
             model = dict(title=str(iade))
             self.set_status(400)
-        except ValueError as ve:
-            model = dict(title=str(ve))
-            self.set_status(400)
-        except Exception as e:
-            model = dict(title=f'Could not parse {path}: {str(e)}')
+            raise web.HTTPError(400, str(iade)) from iade
+        except Exception:
+            model = {"env_list": {}, "inputs": {}, "outputs": {}}
```

Comment:
Could you add a comment here that we're intentionally returning the empty model in this circumstance?
Also, I think logging at least a debug level here might be helpful but indicate that it's being ignored.  In this case, I think it would be good to cast the exception as a string so as to not produce a traceback in the log - since those can look alarming.  This way, we can see if there truly are other cases that should produce more explicit exceptions, etc.